src/metabase/public_settings.clj

```diff
@@ -276,7 +276,7 @@
   (deferred-tru "Allow persisting models into the source database.")
   :type       :boolean
   :default    false
-  :visibility :authenticated)
+  :visibility :public)
```

Comment:
@noahmoss is it safe to expose this flag publicly? Looks safe to me but I'd like to double-check. It is needed for the FE so that the value is provided in the HTML template and routes can be initialized correctly

Reply:
It also seems safe to me. But maybe double check with @luizarakaki

Reply:
Agree. It is fine